workspaces/ui-v2/Taskfile.yml

```diff
@@ -0,0 +1,87 @@
+version: "3"
+
+tasks:
+  clean:
+    desc: Removes UI's build directories
+    cmds:
+      - rm -rf build
+
+  ci:
+    - task: cloud_spec_viewer:ci
+
+  #
+  # docker namespace
+  #
+  docker:build:
+    desc: Package site as a Docker container
+    vars:
+      TAG:
+        sh: echo $(../../bin/clean-docker-tag {{.TAG}})
+    cmds:
+      - |
+        docker build . \
+          -t $DOCKER_REGISTRY/{{.REPO}}:{{.TAG}} \
+          --build-arg APP={{.REPO}}
+
+  docker:push:
+    vars:
+      TAG:
+        sh: echo $(../../bin/clean-docker-tag $TAG)
+    cmds:
+      - docker push $DOCKER_REGISTRY/{{.REPO}}:{{.TAG}}
+      - |
+        if [[ "{{.TAG}}" =~ ^release-b ]]
+        then
+          echo "tagging and pushing 'latest'"
+          docker tag $DOCKER_REGISTRY/{{.REPO}}:{{.TAG}} $DOCKER_REGISTRY/{{.REPO}}:latest
+          docker push $DOCKER_REGISTRY/{{.REPO}}:latest
+        fi
+
+  #
+  # cloud_spec_viewer namespace
+  #
+  cloud_spec_viewer:build:
+    desc: Build cloud_spec_viewer site
+    cmds:
+      - task: :workspaces:build
+        vars:
+          CI: "false"
+      - >
+        BUILD_PATH=build-cloud_spec_viewer
+        CI=false
+        yarn build:cloud
+    sources:
+      - src/**
+      - package.json
+      - '**.ts'
+      - scripts/*spec*
+    generates:
+      - build-cloud_spec_viewer/**
+
+  cloud_spec_viewer:docker:build:
+    - task: cloud_spec_viewer:build
+    - task: docker:build
+      vars:
+        REPO: cloud_spec_viewer
+
+  cloud_spec_viewer:docker:push:
+    vars:
+      TAG:
+        sh: echo $(../../bin/clean-docker-tag $TAG)
+    cmds:
+      - task: docker:push
+        vars:
+          REPO: cloud_spec_viewer
+      - task: :docker:slack-notifier
+        vars:
+          TAG: "{{.TAG}}"
+          REPO: cloud_spec_viewer
```

Comment:
if you're pushing to a new repo, you need to create the repo first. just need to add the desired name to [this](https://github.com/opticdev/monorail/blob/master/projects/terraform/production/ecr/variables.tf) and open a pr.

Reply:
Excellent, love this process :D 

https://github.com/opticdev/monorail/pull/237